README.md

```diff
@@ -74,3 +74,10 @@ Know how to code and you'd like to add a feature? Feel free to [create a PR](htt
 ## Bug Reports
 Spot an issue or bug? Please let us know by [creating a bug report](https://github.com/alec-kr/F1PyStats/issues/new/choose) :bug:
 
+## OUR VALUABLE CONTRIBUTORS✨
```

Comment:
Do you want this heading to be in all capital letters? The rest of the document headings have each first letter capitalized.

Reply:
I will change it to the requested font